### PR TITLE
new page re the 3 integration approaches

### PR DESCRIPTION
**Pull request by genwhittTTD**
New standalone page covering client-side, client-server, server-side, at a high level.

### CODE REVIEW

File: docs/ref-info/ref-integration-approaches.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+- [UID2 Client-Side Integration Guide for Prebid.js](../guides/integration-prebid-client-side.md)
+- [Client-Side Integration Guide for JavaScript](../guides/integration-javascript-client-side.md)
+- [UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)
+
+Advertisers integrating on the client side can use the JavaScript SDK:
+
+- [Client-Side Integration Guide for JavaScript](../guides/integration-javascript-client-side.md)
+
+### Client-Side Integration: Security Values
```
Comment (sunnywu) [inline docs/ref-info/ref-integration-approaches.md:39]: We made up a term for Subscirption ID + Public Key - _Client Keypair_ but we never used this term on doc site (it exists in source code and internal doc only).

_If you choose a client-side integration, you'll get a client keypair which consists of two values that identify you to the UID2 servers: **Subscription ID** and **Public Key**._

This makes sense?

File: docs/ref-info/ref-integration-approaches.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+For example:
+
+- Publishers generate UID2 tokens on the client side for bidstream use, as well as refreshing the tokens.
+- Advertisers generate UID2 tokens on the client side for tracking pixels.
+
+Examples of documentation for publisher client-side integrations:
+
+- [UID2 Client-Side Integration Guide for Prebid.js](../guides/integration-prebid-client-side.md)
+- [Client-Side Integration Guide for JavaScript](../guides/integration-javascript-client-side.md)
```
Comment (sunnywu) [inline docs/ref-info/ref-integration-approaches.md:32]: off-topic - should this be renamed to **UID2** Client-Side Integration Guide for Javascript to keep it consistent? I think originally i suggested to remove the term UID2 but now it seems to be the odd one out...

**Review by sunnywu — APPROVED** [review]

File: docs/ref-info/ref-integration-approaches.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/ref-info/ref-integration-approaches.md:1]: @sunnywu Definitely! Thx much. Done.

File: docs/ref-info/ref-integration-approaches.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/ref-info/ref-integration-approaches.md:32]: @sunnywu Not sure which way to go on this. Noting this inconsistency for further consideration and a near future fix on a separate Jira.

File: docs/ref-info/ref-integration-approaches.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/ref-info/ref-integration-approaches.md:39]: @sunnywu it totally does. Not keen on introducing this new term in just one new doc though. it should be on the Credentials page (which, if you have an idea for a new name, LMK).
Adding it, but will reach out to you and we need to do a bit more on this.

File: docs/ref-info/ref-integration-approaches.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Examples of documentation for publisher client-server integrations:
+
+- [UID2 Client-Server Integration Guide for Prebid.js](../guides/integration-prebid-server-side.md)
+- [Client-Server Integration Guide for JavaScript](../guides/integration-javascript-client-server.md)
+- [UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)
+
+### Client-Server Integration: Credentials
+
+If you choose a client-server integration, you'll need to have an **API key** and **client secret**, values that you must store securely.
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/ref-info/ref-integration-approaches.md:64]: Done, thx.

File: docs/ref-info/ref-integration-approaches.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+For example, in a server-side integration:
+
+- Publishers generate UID2 tokens on the server side for bidstream use.
+- Advertisers generate raw UID2s on the server side to be delivered for audience targeting.
+
+An example of documentation for publisher server-side integration is [Publisher Integration Guide, Server-Side](../guides/integration-publisher-server-side.md).
+
+### Server-Side Integration: Credentials
+
+If you choose a client-side integration, you'll need to have an **API key** and **client secret**, values that you must store securely.
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/ref-info/ref-integration-approaches.md:83]: Done, thx.

**Pull request merged**